lua/telescope/actions/init.lua

```diff
@@ -518,6 +518,27 @@ actions.insert_value = function(prompt_bufnr)
   return selection.value
 end
 
+-- Ask user to confirm an action
+-- @param prompt string: The prompt for confirmation
+-- @param default_value string: The default value of user input
+-- @param yes_values table: List of positive user confirmations ({"y", "yes"} by default)
+-- @return bool: Whether user confirmed the prompt
```

Comment:
```suggestion
--- Ask user to confirm an action
---@param prompt string: The prompt for confirmation
---@param default_value string?: The default value of user input
---@param yes_values table?: List of positive user confirmations ({"y", "yes"} by default)
---@return boolean: Whether user confirmed the prompt
```

Small fix for type annotations